test/xapi/track/image_test.rb

```diff
@@ -0,0 +1,15 @@
+require './test/test_helper'
+require 'xapi/track/image'
+
+class ImageTest < Minitest::Test
+  def test_image
+    path  = './test/fixtures/tracks/fake/docs'
+    image = Xapi::Image.new('test.png', path)
+    image_2 = Xapi::Image.new('no_exists.png', path)
+
+    assert_equal true, image.exists?
```

Comment:
The common idiom here would be `assert image.exists?` rather than using assert_equal.